README.md

```diff
@@ -79,15 +78,7 @@ To use web version of this app with your local server:
 
 1. Make [these small changes](https://github.com/bartekpacia/mentorship-backend/commit/5c4336fa615b0a480af196954b715410e1a41ac3) to your local webserver
    to enable CORS
-2. Run local devserver
-
-### Future?
-
-I'd love Systers to accept this project as one of theirs. I do realize it requires
-better documentation and has to be polished a bit, but I really believe these problems
-are easy to solve.
-My dream? I'll be incredibly happy, for example, if students in next year's Google Code-in will continue its development,
-improve it and polish the codebase.
+2. [Run local devserver](https://github.com/anitab-org/mentorship-backend#run-app)
```

Comment:
```suggestion
2. [Run local server](https://github.com/anitab-org/mentorship-backend#setup-and-run)
```
I changed to #setup-run because I plan to have in that section a mention that people can run all those commands and also with docker. So its just so we don't always redirect them to the long steps

Reply:
OK, I'll fix this :)